lib/constable_web/templates/layout/app.html.eex

```diff
@@ -19,33 +19,27 @@
       <header class="main-header">
         <%= link "#Constable", to: "/", class: "logo" %>
 
-        <ul class="navigation right">
-          <li>
-            <%= form_tag search_path(@conn, :show), method: :get do %>
-              <input
-                class="header-search-input"
-                name="query"
-                placeholder="Search announcements..."
-                type="text"
-                value="<%= @conn.params["query"] %>"
-              />
-            <% end %>
-          </li>
-          <li>
-            <%= link to: announcement_path(@conn, :new), class: "button button-small button-secondary button-icon" do %>
-              <img src="/images/icon-plus-white.svg" />
-            <% end %>
-          </li>
-          <li>
-            <%= link to: settings_path(@conn, :show), class: "settings-gravatar remote-append" do %>
-              <img
-                src="<%= gravatar(@conn.assigns.current_user) %>"
-                class="current-user-gravatar"
-              />
-            <% end %>
-          </li>
-        </ul>
+        <%= form_tag search_path(@conn, :show), class: "header-search-form", method: :get do %>
+          <input
```

Comment:
I know this is legacy code, but might be good to add a corresponding label with this guy with a "for" attribute that links up with an id on the search for ~ accessibility ~. You can just hide it visually so it doesn't show up.

Reply:
> legacy code

![sad](https://user-images.githubusercontent.com/22394/38425768-c6197e9c-3982-11e8-97e3-12d74d6ca18f.gif)

Reply:
😂